app/views/wiki/edit.html.erb

```diff
@@ -57,7 +57,7 @@
           <input class="form-control" id="taginput" type="hidden" name="tags" value="<%= params[:tags] %>" />
 
           <div class="form-group">
-            <input tabindex="1" name="title" type="text" class="form-control" id="title" value="<% if @node && @node.latest %><%= @node.latest.title %><% else %><%= params[:title] %><% end %>">
+            <input tabindex="1" name="title" type="text" class="form-control" id="title" maxlength="50" value="<% if @node && @node.latest %><%= @node.latest.title %><% else %><%= params[:title] %><% end %>">
```

Comment:
Hey @kvenkman, while this is great and its what was described on the issue. I would suggest we also add the validation on the active record level.
 Please change this line https://github.com/publiclab/plots2/blob/master/app/models/node.rb#L97 to 
```
  validates :title, presence: true, length: { minimum: 3, maximum: 50 }
```

Reply:
I've commited changes to my branch - does that work?

Reply:
Yep it does..Thanks